pkg/sources/github/github.go

```diff
@@ -1141,16 +1147,75 @@ func (s *Source) processRepoComments(ctx context.Context, repoPath string, trimm
 		if err := s.processIssueComments(ctx, repoInfo, chunksChan); err != nil {
 			return err
 		}
-
 	}
 
 	if s.includePRComments {
-		return s.processPRComments(ctx, repoInfo, chunksChan)
+		if err := s.processPRComments(ctx, repoInfo, chunksChan); err != nil {
+			return err
+		}
+	}
+
+	if s.includeIssues {
+		if err := s.processIssues(ctx, repoInfo, chunksChan); err != nil {
+			return err
+		}
+		if !s.includeIssueComments {
+			if err := s.processIssueComments(ctx, repoInfo, chunksChan); err != nil {
+				return err
+			}
+		}
+	}
+
+	if s.includePRs {
+		if err := s.processPRs(ctx, repoInfo, chunksChan); err != nil {
+			return err
+		}
+		if !s.includePRComments {
+			if err := s.processPRComments(ctx, repoInfo, chunksChan); err != nil {
+				return err
+			}
+		}
 	}
```

Comment:
**Nit (optional):** I like that we're not repeating work here, but it's easier for me to read the code if its laid out like so:

```go
if s.includeIssues || s.includeIssueComments {
    // processIssueComments
}

if s.includeIssues {
    // processIssues
}

if s.includePRs || s.includePRComments {
    // processPRComments
}

if s.includePRs {
    // processPRs
}
```

Another idea - if we make `includeIssues` only for scanning issues (and not comments), we could make the CLI flag configure both for the same overall functionality. This would make the `Source` slightly more flexible in case we ever want to scan only one or the other.. which is probably an unlikely configuration? That's why this is an optional nit comment 😅

Reply:
Updated codebase for the first suggestion. 

For the second suggestion, would users have to add two flags to the CLI? `--issues --issue-comments` ? The general idea (at least in my mind) is to encourage users to only use the `--issues` flag so that Issue Body text and Issue Comments are all scanned. As it stands in production right now, I think users would assume the initial Issue body text is being scanned.